### ISSUE TITLE
About inconsistent testing set between experiment_data.zip and  kp20k.zip!

### ISSUE THREAD

**Issue by Chen-Wang-CUHK** [memray/seq2seq-keyphrase#15]
Dear Rui Meng,
       First, really thank you for the sharing of your code and dataset.
       But I found that the testing set in experiment_data.zip may be inconsistent with the testing set in kp20k.zip.
       For example, I can not find the paper "a feedback vertex set of <digit> degenerate graphs" (the "0.txt" in experiment_data\...\baseline-data\kp20k) in kp20k\kp20k_testing.json .
      Does the inconsistency really exist?
Best,
Wang

**Comment by memray** [comment]
Hi Wang,

Sorry for the late response, and sorry for the mistake. I've updated the data and put it [here](https://drive.google.com/file/d/1ZTQEGZSq06kzlPlOv4yGjbUpoDrNxebR/). I guess the data dump was exported earlier and it is in a different order from the current data as I shuffled it every time. Thankfully I stored the final used sequence on the disk. Hopefully, it was not used for later studies. I'll check it in detail and let you know if any change is needed. Feel free to let me know if there's any other problem.

Thanks,
Rui
